spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/hive/HivePropertiesConstants.java

```diff
@@ -31,12 +31,14 @@ public class HivePropertiesConstants {
   public static final String GRAVITINO_HIVE_FORMAT_CSV = StorageFormat.CSV.toString();
   public static final String GRAVITINO_HIVE_EXTERNAL_TABLE = EXTERNAL_TABLE.name();
   public static final String GRAVITINO_HIVE_TABLE_TYPE = "table-type";
+  public static final String GRAVITINO_HIVE_LOCATION = "location";
```

Comment:
GRAVITINO_HIVE_LOCATION -> GRAVITINO_HIVE_TABLE_LOCATION ?

Reply:
please replace `location` with `HiveTablePropertiesMetadata.LOCATION`

Reply:
`table-type` should replaced by `HiveTablePropertiesMetadata.TABLE_TYPE` could you fix it together?